### ISSUE TITLE
Build for linux/arm/v7?

### ISSUE THREAD

**Issue by cj13579** [Wingysam/Christmas-Community#133]
I'm migrating a bunch of my things to a new QNAP NAS. When pulling down the docker image for the project I get the following:

```
no matching manifest for linux/arm/v7 in the manifest list entries
```

Would it be possible to add `linux/arm/v7` to the [build targets](https://docs.docker.com/build/building/multi-platform/#building-multi-platform-images)?

**Comment by cj13579** [comment]
@Wingysam do you have some secret sauce on the machine where you do your docker builds? I can't get the images to build unless I make changes to the Dockerfile. The error that I am getting is:

```
sh: tsc: not found
```

I'm happy to create a PR to modify the release command if that's OK?

**Comment by Wingysam** [comment]
Yeah tsc is only in devDependencies and the Dockerfile skips them. I have no idea how that ever worked. node_modules is in the dockerignore so it couldn't be that.

Yes I would merge a PR. Otherwise, I can fix it when I work on the project next.
